src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -221,8 +235,13 @@ const JobHistory = _.flow(
                 headerRenderer: () => h(HeaderCell, ['Status']),
                 cellRenderer: ({ rowIndex }) => {
                   const { workflowStatuses, status } = filteredSubmissions[rowIndex]
+                  const collapsedStatusList= collapsedStatuses(workflowStatuses)
+                  const statusString =
+                        'failed' in collapsedStatusList ? `Failed` :
+                          'abort' in collapsedStatusList ? 'Aborted' :
+                            status
```

Comment:
@panentheos what do you think is the clearest way to do this?

Reply:
This is clear. We can change it later if we come up with some pattern that is obviously better, but this is quite nice.